src/bootstrap.php

```diff
@@ -1812,6 +1829,89 @@ function (Accept $type, string $id) use ($app) {
     'application/vnd.elife.profile+json; version=1'
 ));
 
+$app->get('/promotional-collections', function (Request $request, Accept $type) use ($app) {
+    $promotionalCollections = $app['promotional-collections'];
+
+    $page = $request->query->get('page', 1);
+    $perPage = $request->query->get('per-page', 10);
+    $subjects = (array) $request->query->get('subject', []);
+    $containing = (array) $request->query->get('containing', []);
+
+    if (false === empty($subjects)) {
+        $promotionalCollections = array_filter($promotionalCollections, function ($promotionalCollection) use ($subjects) {
+            $promotionalCollectionsSubjects = array_map(function (array $subject) {
+                return $subject['id'];
+            }, $promotionalCollection['subjects'] ?? []);
+
+            return count(array_intersect($subjects, $promotionalCollectionsSubjects));
+        });
+    }
+
+    if (false === empty($containing)) {
+        $promotionalCollections = array_filter($promotionalCollections, function ($promotionalCollection) use ($containing) {
```

Comment:
I assume this only applies to `content` and not to:
- `relatedContent` (as it's kind of a suggestion for other content, but not contained in here)
- `podcastEpisodes` (which are managed separately)

Reply:
This is the same way that it is implemented with collections.

Reply:
Yes, just stating it out loud to see if I was mistaken.